packages/veritone-widgets/src/redux/modules/engineSelection/index.js

```diff
@@ -294,7 +375,7 @@ export function initializeWidget(id) {
   };
 }
 
-export function refetchEngines(id) {
+export function refetchEngines(id, currentTabIndex) {
```

Comment:
why does this need an ID? relatedly, enginesModule.fetchEngines doesn't even use the ID you have it accepting...

Reply:
you're right, I added this by mistake, will fix.